docs/getting-started-on-snowplow-open-source/quick-start-aws/send-test-events-to-your-pipeline/index.md

```diff
@@ -4,11 +4,11 @@ date: "2021-07-08"
 sidebar_position: 100
 ---
 
-Now your pipeline is up and running, we can send a simple page_view event to it to validate that it is working.
+Now your pipeline is up and running, we can send a simple `page_view` event to it to validate that everything is working.
 
 ##### Using cURL
```

Comment:
I think we should discourage cURL — I see a lot of discourse posts about invalid data that turn out to be because of this. Can we recommend the cli tracker instead?

Reply:
Great idea!  It might even be better if we have a live app website or something where they can put in their collector endpoint and just generate a bunch of dummy data?